frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -299,6 +300,7 @@ export default {
       preferredContentTypes: () => this.preferredContentTypes,
       allContentNodes: () => this.contentNodes,
       camp: () => this.camp,
+      isPaperDisplaySize: computed(() => this.isPaperDisplaySize),
```

Comment:
Why is the computed needed? Shouldn't `isPaperDisplaySize` already be reactive?

Reply:
Non-object injected properties are not reactive: https://vuejs.org/guide/components/provide-inject#working-with-reactivity

Reply:
Ah I see, thanks for clarifying